zeppelin-server/src/test/java/org/apache/zeppelin/rest/AbstractTestRestApi.java

```diff
@@ -243,14 +253,25 @@ protected static PostMethod httpPost(String path, String body) throws IOExceptio
     return postMethod;
   }
 
-  protected Matcher<GetMethod> responsesWith(final int expectedStatusCode) {
-    return new TypeSafeMatcher<GetMethod>() {
-      WeakReference<GetMethod> method;
+  protected static PutMethod httpPut(String path, String body) throws IOException {
+    LOG.info("Connecting to {}", url + path);
+    HttpClient httpClient = new HttpClient();
+    PutMethod putMethod = new PutMethod(url + path);
```

Comment:
same here:
you will need to add the origin header:
PutMethod.addRequestHeader("Origin", "http://localhost");